bigtable/tests/unit/test_row.py

```diff
@@ -78,80 +78,65 @@ def _make_client(self, *args, **kwargs):
         return self._get_target_client_class()(*args, **kwargs)
 
     def test_constructor(self):
-        row_key = b'row_key'
-        table = object()
-
-        row = self._make_one(row_key, table)
-        self.assertEqual(row._row_key, row_key)
-        self.assertIs(row._table, table)
-        self.assertEqual(row._pb_mutations, [])
+        from google.cloud.bigtable_v2.proto import bigtable_pb2
 
-    def test_constructor_with_unicode(self):
-        row_key = u'row_key'
-        row_key_bytes = b'row_key'
+        row_key = b'row_key'
         table = object()
 
         row = self._make_one(row_key, table)
-        self.assertEqual(row._row_key, row_key_bytes)
+        self.assertEqual(row.row_mutations.mutations_entry.row_key, row_key)
         self.assertIs(row._table, table)
+        entry = bigtable_pb2.MutateRowsRequest.Entry()
+        self.assertEqual(row.row_mutations.mutations_entry.mutations,
+                         entry.mutations)
 
     def test_constructor_with_non_bytes(self):
         row_key = object()
         with self.assertRaises(TypeError):
             self._make_one(row_key, None)
 
-    def test__get_mutations(self):
+    def test_row_mutations(self):
         row_key = b'row_key'
         row = self._make_one(row_key, None)
 
-        row._pb_mutations = mutations = object()
-        self.assertIs(mutations, row._get_mutations(None))
+        row._row_mutations = mutations = object()
+        self.assertIs(mutations, row.row_mutations)
 
-    def _set_cell_helper(self, column=None, column_bytes=None,
-                         value=b'foobar', timestamp=None,
-                         timestamp_micros=-1):
-        import six
-        import struct
+    def _set_cell_helper(self, column=None, value=b'foobar', timestamp=None):
+
+        from google.cloud.bigtable.mutation import RowMutations
 
         row_key = b'row_key'
-        column_family_id = u'column_family_id'
+        column_family_id = 'column_family_id'
```

Comment:
The `u''` prefix ensures that the value is "text" on Python2, rather than bytes.

Reply:
@aneepct: ping